src/pages/blogpost/Blogpost.jsx

```diff
@@ -0,0 +1,60 @@
+import { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { useNavigate } from 'react-router-dom';
+import axios from 'axios';
+
+// Helpers
+import formatDate from '../../helpers/formatDate';
+
+// Components
+import Button from '../../components/button/Button';
+import CommentsAndShares from '../../components/comments-and-shares/CommentsAndShares';
+
+// Style
+import './blogpost.css';
+
+const Blogpost = () => {
+    const navigate = useNavigate();
+    // Find blogpost
+    const { id } = useParams();
+    const [singlePost, setSinglePost] = useState({});
```

Comment:
Ik zou hier een default waarde `undefined` gebruiken. Dan is het verschil of het al wél of niet is opgehaald wat duidelijker en kan je check verderop iets simpeler
```suggestion
    const [singlePost, setSinglePost] = useState();
```